### ISSUE TITLE
Option for multiple highlight styles instead of one

### ISSUE THREAD

**Issue by IHateMyLife188** [chetachiezikeuzor/Highlightr-Plugin#85]
Is there a way where I can use more than 1 highlight style? There are times where I want to use the rounded style but there are also times where I have to use floating for some specific texts

**Comment by 0x7969** [comment]
Yes that'd be great, I'd even love an option to provide custom css properties to each named highlight-style and not just the color. But being able to choose color plus one of your predefined highlight-styles would be great aswell. Thanks for your work btw!
